crates/miden-objects/src/account/account_id/network_id.rs

```diff
@@ -1,20 +1,60 @@
-use alloc::string::ToString;
+use alloc::string::{String, ToString};
 use core::str::FromStr;
 
 use bech32::Hrp;
 
 use crate::errors::NetworkIdError;
 
+/// A wrapper around HRP for custom network identifiers.
+#[derive(Debug, Clone, PartialEq, Eq, PartialOrd, Ord)]
+pub struct CustomHrp {
```

Comment:
```suggestion
pub struct CustomNetworkId {
```

I think I'd call it a custom network ID.